### ISSUE TITLE
plots should have a minimum resize level

### ISSUE THREAD

**Issue by Seanny123** [nengo/nengo_gui#547]
When resizing a plot, you can resize it so that it becomes invisible by resizing to the extreme top right corner. At that point, only the label remains visible, consequently the plot cannot be resized any further, nor can it be moved.

There are a few options to solve this:
- Make a minimum plot resize level
- Make the label selectable when the plot starts to have negative dimensions

**Comment by jgosmann** [comment]
With a minimum size the plots might grow in size when zooming. If you zoom out and a plot hits it's minimum size, it would stay that size while zooming further out. Once you zoom back in it starts growing, but the relative size to other plots is now larger.

**Comment by Seanny123** [comment]
Good point, I hadn't considered that! So then the first option is really the only option then, without a bunch of awkward resize memory coding?

**Comment by tcstewar** [comment]
> Good point, I hadn't considered that! So then the first option is really the only option then, without a bunch of awkward resize memory coding?

We have to be really careful here with minimum sizes.  In particular, these should be minimum *displayed* sizes, not a minimum that affects the actual stored size in the .cfg file.  Basically, we don't want a situation where if I zoom out and then zoom in again, it changes the size of the plot.  

Also, remember that the label isn't always visible, so that can't be the fall-back.

I believe a decent solution would be to have a minimum plot resize level, but that it's pixel-based, and only influences things when dragging the edges of a plot.  It should also be fairly small, so that it's still possible to make the plots very tiny.  Just not so tiny that it's impossible to expand them again.
